python/lsst/ctrl/bps/bps_utils.py

```diff
@@ -58,12 +62,16 @@ def chdir(path):
         os.chdir(cur_dir)
 
 
-def create_job_quantum_graph_filename(job, out_prefix=None):
+def create_job_quantum_graph_filename(config, job, out_prefix=None):
     """Create a filename to be used when storing the QuantumGraph
     for a job.
 
     Parameters
     ----------
+    job : `lsst.ctrl.bps.GenericWorkflowJob`
```

Comment:
It looks like this line should be removed. The parameter `job` is the second parameter in function signature and it is properly described later.